components/clickup/actions/create-chat-view-comment/create-chat-view-comment.mjs

```diff
@@ -71,15 +25,28 @@ export default {
     },
     assignees: {
       propDefinition: [
-        clickup,
+        common.props.clickup,
         "assignees",
         (c) => ({
           workspaceId: c.workspaceId,
         }),
       ],
       optional: true,
     },
+    listWithFolder: {
+      type: "boolean",
+      label: "List With Folder?",
+      description: "If `true`, the list will be filtered by Folder ID",
+      reloadProps: true,
+      optional: true,
+    },
```

Comment:
Can `listWithFolder` be a propDefinition, or imported from a common file? Since it's being used across most of the actions and the label, description etc. are all duplicated